### PR TITLE
{lang}[foss-2016a] Clang 3.7.1 (REVIEW)

### PR DESCRIPTION
**Pull request by hajgato**

### CODE REVIEW

File: easybuild/easyconfigs/c/Clang/Clang-3.7.1-foss-2016a.eb [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+sources = [
+    "llvm-%(version)s.src.tar.xz",
+    "cfe-%(version)s.src.tar.xz",
+    "compiler-rt-%(version)s.src.tar.xz",
+    "polly-%(version)s.src.tar.xz",
+]
+
+dependencies = [
+    ('GMP', '6.1.0'),
+    ('ISL', '0.15'),
```
Comment (boegel) [inline easybuild/easyconfigs/c/Clang/Clang-3.7.1-foss-2016a.eb]: ISL 0.15 dates from 201506, there's an ISL 0.17 already, why not use that?

similar for GMP: 6.1.1

Comment (hajgato) [inline easybuild/easyconfigs/c/Clang/Clang-3.7.1-foss-2016a.eb]: @boegel done

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 2 out of 3 (3 easyconfigs in this PR)
node2110.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/dd106e82f87bcc7854045d3bf8f167d3 for a full test report.

**Comment by boegel** [comment]
@hajgato ah, Python also has GMP as a dep, so you'll need to stick with 6.1.0 after all...

(see `eb --check-conflicts`)

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 2 out of 3 (3 easyconfigs in this PR)
node2503.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/df0f1e2ff0a46766ca1e6373fefc3d28 for a full test report.

File: easybuild/easyconfigs/i/ISL/ISL-0.15-foss-2016a.eb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+homepage = 'http://isl.gforge.inria.fr/'
+description = """isl is a library for manipulating sets and relations of integer points bounded by linear constraints."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = ['http://isl.gforge.inria.fr/']
+sources = [SOURCELOWER_TAR_BZ2]
+
+dependencies = [('GMP', '6.1.0')]
```
Comment (wpoely86) [inline easybuild/easyconfigs/i/ISL/ISL-0.15-foss-2016a.eb]: use same version as in the Clang easyconfig

Comment (wpoely86) [inline easybuild/easyconfigs/i/ISL/ISL-0.15-foss-2016a.eb]: a wait, why is this one here? Only version 0.17 is used?

Comment (boegel) [inline easybuild/easyconfigs/i/ISL/ISL-0.15-foss-2016a.eb]: original PR was using 0.15, it doesn't hurt to keep this one around too

**Comment by hajgato** [comment]
@boegel Back to `GMP 6.1.0`

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2159.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/2666b1926a9f3332794d3307e14112cf for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2408.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/9b988d00ebc8f93e2c111219a88f4552 for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Going in, thanks @hajgato!

**Pull request merged**